src/components/Modals/NewFileModal.js

```diff
@@ -59,15 +64,18 @@ const NewFileModal = () => {
       </div>
 
       <div className='modal-content__body'>
-        <label htmlFor='name'>{t('filePane.newFileModal.inputLabel')}</label>
+        <div>
+          <label htmlFor='name'>{t('filePane.newFileModal.inputLabel')}</label>
+          <p className='modal-content__help-text'>{t('filePane.newFileModal.helpText', {examples: FILE_NAME_EXAMPLES[projectType]})}</p>
```

Comment:
nothing wrong here, but an alternative might have been to use a template literal, so you could have for example used:
```
t('filePane.newFileModal.helpText', {examples: `filePane.newFileModal.${projectType}HelpTextExample`})
```
obvs changing the key accordingly, or uppercasing the first letter and using as is (don't know if we use lodash, but that has a handy `_.upperFirst()` method)

Reply:
Going to throw my 2 cents in because I was thinking something similar 😅 you could restructure the i18n text to something like this
```
helpTextExample: {
  html: "'file.html' or 'file.css'",
  python: "'file.py'",
```
and make do a similar string interpolation Dan suggested without fiddling the `projectType` 😄

Reply:
nice! that's a tidier approach

Reply:
Ahh this sounds sensible - did feel what I was doing was somewhat clunky...